client/src/pages/Search.jsx

```diff
@@ -64,6 +69,7 @@ function Search(props) {
       city: city,
       bike_friendly: bikeR,
       pet_friendly: petR,
+      top_rated_landlords: topL,
```

Comment:
consider taking all `top_rated_landlords` logic and creating a new feature branch specifically for this.